readme.md

```diff
@@ -173,6 +169,14 @@ Once you do a search, you should see a list similar to the one below (versions m
 Please use [GitHub Issues](https://github.com/windows-toolkit/WindowsCommunityToolkit/issues) for bug reports and feature requests.
 For general questions and support, please use [Stack Overflow](https://stackoverflow.com/questions/tagged/windows-community-toolkit) where questions should be tagged with the tag `windows-community-toolkit`.
 
+## <a name="supported"></a> Required Dependencies
+The following dependencies are required for building the Windows Community Toolkit repo and sample app:
+
+* [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
+* [Windows SDK May 2019 Update	18362](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive)
+* [.NET Core 3.0.100 SDK](https://dotnet.microsoft.com/download/visual-studio-sdks)
+* [.NET Framework 4.6.1 Developer Pack](https://dotnet.microsoft.com/download/visual-studio-sdks)
```

Comment:
It should be _.NET Framework 4.6.**2** Developer Pack_
![image](https://user-images.githubusercontent.com/24755596/75033917-04494f00-54d2-11ea-82a4-fac26bfecb96.png)
```suggestion
* [.NET Framework 4.6.2 Developer Pack](https://dotnet.microsoft.com/download/visual-studio-sdks)
```